postgres/9.6/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM postgres:9.6
+
+MAINTAINER Debezium Community
+
+# Install the packages which will be required to get everything to compile
+RUN apt-get update && apt-get install -y build-essential pkg-config git postgresql-server-dev-9.6 software-properties-common
+
+# The plugin requires protobuf-c at least 1.1 but the stable Debian repo has an older version, so we need to add the Debian testing repo which has the correct version
+RUN add-apt-repository "deb http://ftp.debian.org/debian testing main contrib" && apt-get update
+ 
+# Install the protobuf-c dependency for the plugin
+RUN apt-get install -y libprotobuf-c-dev=1.2.1-1+b1
```

Comment:
You should probably run `rm -rf /var/lib/apt/lists/*` to clean up the `apt-get` cache populated by the multiple `apt-get` calls. But because of layering, cleaning the cache is only affective for files added to the cache during that one `RUN` statement, so for this reason you actually should combine all three of these `RUN` statements into a single one. This way, the resulting layer has only the dependencies that are needed below.

Technically, you could combine all of five of the `RUN` commands in a single one, but there are a couple of reasons I'd suggest not doing this. First, the first three existing `RUN` commands install dependencies, while the last two build and install the decoderbufs library into postgres, so it's useful to keep this distinction. Second, if the layer resulting from installing the dependencies were minimized, and the layer resulting from building and installing decoderbufs is minimized, combining these into an "uber" layer is not more efficient. And third, keeping a separate layers means that, when you build locally, Docker can cache the layers and will only have to build them if something changes -- something that's not frequent for the decoderbufs layer and even less likely for the dependency layer.